packages/docusaurus-theme-classic/src/theme/prism-include-languages.ts

```diff
@@ -26,6 +26,10 @@ export default function prismIncludeLanguages(
   globalThis.Prism = PrismObject;
 
   additionalLanguages.forEach((lang) => {
+    if (lang === 'php') {
+      // eslint-disable-next-line global-require
+      require('prismjs/components/prism-markup-templating.js');
+    }
```

Comment:
wondering why do we need this now?

Reply:
Not sure why it's not an issue previously, but the error is mentioned here: https://github.com/PrismJS/prism/issues/1400#issuecomment-485847919